framework/16-inch/cpu/common/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, pkgs, ... }: {
+  imports = [
+    ../../../../common/pc/laptop
+    ../../../../common/pc/laptop/ssd
+  ];
+
+  # Fix TRRS headphones missing a mic
+  # https://community.frame.work/t/headset-microphone-on-linux/12387/3
+  boot.extraModprobeConfig = lib.mkIf (lib.versionOlder pkgs.linux.version "6.6.8") ''
+    options snd-hda-intel model=dell-headset-multi
+  '';
+
+  # For fingerprint support
+  services.fprintd.enable = lib.mkDefault true;
+
+  # Custom udev rules
+  services.udev.extraRules = ''
+    # Ethernet expansion card support
+    ACTION=="add", SUBSYSTEM=="usb", ATTR{idVendor}=="0bda", ATTR{idProduct}=="8156", ATTR{power/autosuspend}="20"
+  '';
```

Comment:
```suggestion
  # Custom udev rules
  services.udev.extraRules = ''
    # Ethernet expansion card support
    ACTION=="add", SUBSYSTEM=="usb", ATTR{idVendor}=="0bda", ATTR{idProduct}=="8156", ATTR{power/autosuspend}="20"
    
    # LED Matrix, ModemManager should ignore
    # Required for ledmatrix.frame.work to work
SUBSYSTEMS=="usb", ATTRS{idVendor}=="32ac", ATTRS{idProduct}=="0020", TAG+="uaccess", ENV{ID_MM_DEVICE_IGNORE}="1"
  '';
```

Reply:
This probably shouldn't be added in my pull request because I don't want to have the LED modules to test